RELEASES.md

```diff
@@ -0,0 +1,60 @@
+# Release cycle and process
+Velero team is aiming for 3 releases per year, but will commit to two releases per year with few patch releases in between.
+Planing for new release starts with relabeling all candidates that were not completed in the current release - in example [1.11-candidates](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A1.11-candidate) and labeling all the new for the upcoming release.
+
+In parallel we start a [discussion](https://github.com/vmware-tanzu/velero/discussions?discussions_q=label%3ASurvey) to collect new ideas, proposals, needs and new contributors.
+
+# Versioning and Release
+This document describes the versioning and release process of Velero. This document is a living document, contents will be updated according to each releases.
+
+## Releases
+Velero releases will be versioned using dotted triples, similar to [Semantic Version](http://semver.org/). For this specific document, we will refer to the respective components of this triple as `<major>.<minor>.<patch>`. The version number may have additional information, such as "-rc1,-rc2,-rc3" to mark release candidate builds for earlier access. Such releases will be considered as "pre-releases".
+
+### Major and Minor Releases
+Major and minor releases of Velero will be branched from `main` when the release reaches to `RC(release candidate)` state. The branch format should follow `release-<major>.<minor>.0`. For example, once the release `v1.0.0` reaches to RC, a branch will be created with the format `release-1.0.0`. When the release reaches to `GA(General Available)` state, The tag with format `v<major>.<minor>.<patch>` and should be made with command `git tag -s v<major>.<minor>.<patch>`. The release cadence is around 3 months, might be adjusted based on open source event, but will communicate it clearly.
```

Comment:
The current branch format is `release-1.0' since all the related patch release for that minor release will be in this branch.